.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+  schedule:                # This trigger is here only to make sure
+    - cron: '17 0 * * SUN' # the example itself doesn't become stale
```

Comment:
The comment indicates that the cron isn't needed for the project 😛

Reply:
Yes, that is not needed. I overlooked it. Thanks.